bin/heudiconv

```diff
@@ -26,8 +26,40 @@ import tarfile
 import dicom as dcm
 import dcmstack as ds
 
+from collections import namedtuple
+from os.path import isdir
+
 import logging
 lgr = logging.getLogger('heudiconv')
+# Rudimentary logging support.  If you want it better -- we need to have
+# more than one file otherwise it is not manageable
+logging.basicConfig(
+    format='%(levelname)s: %(message)s',
+    level=getattr(logging, os.environ.get('HEUDICONV_LOGLEVEL', 'INFO'))
+)
+lgr.debug("Starting the abomination")  # just to "run-test" logging
+
+
+SeqInfo = namedtuple(
```

Comment:
eventually I might consider using attrs module to use here (https://attrs.readthedocs.io/en/stable/) instead of namedtuple